.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,19 +1,17 @@
 ---
 name: Bug report
 about: Create a report to help us improve
-
+label: Bug
 ---
 
 **Describe the bug**
-A clear and concise description of what the bug is.
+<!-- A clear and concise description of what the bug is. -->
 
 **To Reproduce**
-Code to reproduce the behavior:
-
+<!-- Code to reproduce the behavior: -->
```

Comment:
"Please include the results of `conda list` and `python -c "import qcelemental as qc; print(qc.__file__, qc.__version__)"`.